config/teleop.yaml

```diff
@@ -49,4 +49,6 @@ teleop:
     twist: 2
     dampen: 3
     pan: 4
-    tilt: 5
\ No newline at end of file
+    tilt: 5
+  topic_echo_presets:
+    esw: ["/joystick"]
```

Comment:
This looks to be a feature for ROS tool presets, should probably remove it from this PR.